pages/1.12/release-notes/1.12/index.md

```diff
@@ -0,0 +1,264 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.12.0
+title: Release Notes for 1.12.0 
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.12.0
+---
+
+DC/OS 1.12.0 was released on October 26, 2018.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.12/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+# About DC/OS 1.12.0 
+
+DC/OS 1.12.0 includes the following new features and capabilities:
+
+## New Features and Capabilities
+
+### Mesosphere Kubernetes Engine
+- High density [Multi-Kubernetes](https://docs.mesosphere.com/services/beta-kubernetes/2.0.0-1.12.0-beta/) leveraging enhanced containerized bin packing. 
+
+### Mesosphere Jupyter Service (MJS)
+- Delivered secure, cloud-native Jupyter Notebooks-as-a-Service to empower data scientists to perform analytics and distributed machine learning on elastic GPU-pools with access to big and fast data services.
+- Secured connectivity to data lakes and data sets on S3 and (Kerberized) HDFS.
+- GPU-enabled Spark and distributed TensorFlow.
+- OpenID connect authentication and authorization with support for Windows Integrated Authentication (WIA) and Active Directory Federation Services (ADFS)
+
+### Observability - Metrics
+Maturation of [metrics](https://docs.mesosphere.com/1.12/metrics/) observability are:
+- Adopted industry standards, Free and Open Source Software (FOSS) metrics infrastructure.
+- Used flexible and configurable metrics pipeline with multiple output formats.
+- Enhanced support for application metric types including histograms, counters, timers, and gauges.
+- Support for sample rates and multi-metrics packets. 
+- Mesos framework metrics are now [available](http://mesos.apache.org/documentation/latest/monitoring/#frameworks).
+- No longer require modifications when collecting metrics via Prometheus endpoint in 1.11.
+
+[enterprise]
+### Private Package Registry
+[/enterprise]
+- Enabled [on-premise package distribution and management](https://docs.mesosphere.com/1.12/administering-clusters/repo/package-registry/).
+- Enabled air-gapped Virtual Private Cloud package management.
+- Simplifies package artifact management.
+- Support for NFS and S3 as storage backends for package storage.
```

Comment:
We don't support NFS Storage backend. @anithamuthu79 can you guide me on how this information is curated?

Reply:
@takirala Pls refer the gDoc that @yishan-lin had shared https://docs.google.com/document/d/1xv-sq_mwG96GnvmbxQxiULPkvO5aVRKFqny-3hHpnN4/edit#heading=h.1w8qtj5xgzg.

Reply:
Thanks for the links!

@somikbehera just as fyi, we don't support NFS storage.
@anithamuthu79  please remove the NFS as a supported storage.

Reply:
Done. Ty @takirala.